### ISSUE TITLE
`latexminted`: not found error when building file with `minted` package

### ISSUE THREAD

**Issue by vstollen** [aergus/dockerfiles#16]
While compiling a document using the `minted` package, I received the following error:

```
[18]sh: 1: latexminted: not found
! Undefined control sequence.
<recently read> \@firstofttwo 
              
l.57 ... data is mapped to \mintinline{java}{byte[]}
                                                   and structs are mapped to
? 
! Emergency stop.
<recently read> \@firstofttwo 
              
l.57 ... data is mapped to \mintinline{java}{byte[]}
                                                   and structs are mapped to
```

I was able to fix it by running `apt install texlive-latex-extra` before compiling. However, I don't understand why the `latexminted` command is not installed with `texlive-full` package.

In case it's relevant, I compile my file using the following command:
```console
$ latexmk -lualatex -latexoption='-shell-escape' "main.tex"
```

**Comment by vstollen** [comment]
I've created a fork where the `texlive-latex-extra` package is installed: https://github.com/vstollen/docker-latex

There, I've also published the image for easy use:
```console
docker pull ghcr.io/vstollen/docker-latex:master
```

If you think including the `texlive-latex-extra` package makes sense, I can create a pull request (or you can add it yourself, if that's quicker).

**Comment by aergus** [comment]
`texlive-latex-extra` is indeed a dependency of [`texlive-full`](https://packages.debian.org/trixie/texlive-full) and hence is installed in the image.

However, I can imagine that the issue is present in the specific version `2024.20241102-1` installed in the image `aergus/latex:2024-11-17`and was fixed in the newer version `2024.20241115-1` that was probably installed when you installed `texlive-latex-extra` explicitly at a later date.

I have pushed a fresh build of the image, `aergus/latex:2024-11-27`. Does the problem persist with this image?
